Sources/StreamChatUI/Composer/ChatMenu/EmojiMenu/SendEmojiViewController.swift

```diff
@@ -0,0 +1,134 @@
+            var sendStickerParam = [String: RawJSON]()
+            sendStickerParam["giftPackageId"] = .string(self.packageInfo?.packageID?.string ?? "")
+            sendStickerParam["giftPackageName"] = .string(self.packageInfo?.packageName ?? "")
+            sendStickerParam["giftPackageImage"] = .string(self.packageInfo?.packageImg ?? "")
+            sendStickerParam["giftSenderId"] = .string(ChatClient.shared.currentUserId?.string ?? "")
+            sendStickerParam["giftSenderName"] = .string(ChatClient.shared.currentUserController().currentUser?.name ?? "")
+            sendStickerParam["channelId"] = .string(cid.description)
+            sendStickerParam["giftReceiverId"] = .string(chatMembers.id ?? "")
+            sendStickerParam["giftReceiverName"] = .string(chatMembers.name ?? "")
+            ChatClient.shared.channelController(for: cid)
+                .createNewMessage(
+                    text: "",
+                    pinning: nil,
+                    attachments: [],
+                    extraData: ["sendStickerGift": .dictionary(sendStickerParam)],
+                    completion: nil)
+            self.view.window?.rootViewController?.dismiss(animated: true, completion: nil)
+        }
+    }
+
+    @IBAction func btnSendSticker(_ sender: Any) {
+        guard let cid = chatChannelController?.channel?.cid else { return }
+        let channelMember = chatChannelController?.client.memberListController(query: .init(cid: cid))
+        channelMember?.synchronize({ [weak self] error in
+            guard error == nil, let self = self else { return }
+            let chatMembers = channelMember?.members.filter({ (member: ChatChannelMember) -> Bool in
+                return member.id != self.chatChannelController?.client.currentUserId
+            })
+            guard let member = chatMembers?.first else { return }
+            self.sendStickerGift(chatMembers: member, cid: cid)
+        })
+    }
+
+    private func loadSticker(stickerId: String) {
+        // Retrieve from userdefault
+        if let stickers = UserDefaults.standard.retrieve(object: [Sticker].self, fromKey: stickerId) {
+            self.stickers = stickers
+            stickerCollectionView.reloadData()
+        } else {
+            StickerApiClient.stickerInfo(stickerId: stickerId) { [weak self] result in
+                guard let `self` = self else { return }
+                self.stickers = result.body?.package?.stickers ?? []
+                // Cache sticker in userdefault
+                UserDefaults.standard.save(customObject: self.stickers, inKey: stickerId)
+                UserDefaults.standard.synchronize()
+                self.stickerCollectionView.reloadData()
+            }
+        }
+    }
+
+    @IBAction func btnCloseAction(_ sender: Any) {
+        self.dismiss(animated: true, completion: nil)
+    }
+}
+
+// MARK: - Collection view delegate
+@available(iOS 13.0, *)
+extension SendEmojiViewController: UICollectionViewDelegate { }
```

Comment:
@mdhanif-simformsolutions  Pls remove UICollectionViewDelegate

Reply:
Updated this thanks.